gapis/gfxapi/vulkan/command_buffer_rebuilder.go

```diff
@@ -0,0 +1,300 @@
+// Copyright (C) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vulkan
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/gapid/gapis/atom"
+	"github.com/google/gapid/gapis/gfxapi"
+
+	"github.com/google/gapid/gapis/memory"
+)
+
+func recreateCmdBeginRenderPass(
```

Comment:
Maybe we can call it rebuildCmdBeginRenderPass, so be consistent with rebuildCommandBuffer, and differentiate from MEC support?